src/schema/v2/Alerts/index.ts

```diff
@@ -178,177 +178,210 @@ export const AlertType = new GraphQLObjectType<
   ResolverContext
 >({
   name: "Alert",
-  fields: () => ({
-    ...IDFields,
-    acquireable: {
-      type: GraphQLBoolean,
-    },
-    additionalGeneIDs: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ additional_gene_ids }) => additional_gene_ids,
-    },
-    additionalGeneNames: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ additional_gene_names }) => additional_gene_names,
-    },
-    artistIDs: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ artist_ids }) => artist_ids,
-    },
-    artists: {
-      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(ArtistType))),
-      resolve: async ({ artist_ids }, _args, { artistsLoader }) => {
-        if (!artist_ids) return []
+  fields: () => {
+    const {
+      filterArtworksConnectionWithParams,
+    } = require("../filterArtworksConnection")
```

Comment:
**question (non-blocking):** Is there a reason this is not an ordinary `import`?

Reply:
It is imported via `require` in all other places. When using `import` it fails with

```
TypeError: Cannot read properties of undefined (reading 'connectionType')
    at FilterArtworksFields (/Users/nikitaskalkin/Projects/metaphysics/src/schema/v2/filterArtworksConnection.ts:267:31)
    at connectionFields (/Users/nikitaskalkin/Projects/metaphysics/src/schema/v2/filterArtworksConnection.ts:376:55)
    at Object.<anonymous> (/Users/nikitaskalkin/Projects/metaphysics/src/schema/v2/filterArtworksConnection.ts:384:8)
```